compiler/il/OMRNode.cpp

```diff
@@ -3359,6 +3359,42 @@ OMR::Node::getOwningMethod()
    }
 
 
+/** \brief
+ *  Checks for AOT flag and identifies owning method using byte code info
+ * 
+ *  \param &bcInfo 
+ * _byteCodeInfo of a node of type TR::Node  
+ * 
+ *  \param *comp  
+ * _compilation of type TR::Compilation
+ * 
+ *  @return 
+ *  Returns object of type TR_OpaqueMethodBlock*
+ */
+TR_OpaqueMethodBlock* 
+OMR::Node::getOwningMethod(TR_ByteCodeInfo &bcInfo, TR::Compilation *comp)
+   {
+   TR_OpaqueMethodBlock *method = NULL; 
+  
+   if (comp->compileRelocatableCode()) 
+      { 
+      if (0 <= bcInfo.getCallerIndex()) 
+         method = (TR_OpaqueMethodBlock *)(((TR_AOTMethodInfo *)comp->getInlinedCallSite(bcInfo.getCallerIndex())._vmMethodInfo)->resolvedMethod->getNonPersistentIdentifier()); 
+      else 
+         method = (TR_OpaqueMethodBlock *)(comp->getCurrentMethod()->getNonPersistentIdentifier()); 
```

Comment:
This doesn't seem right - why would we use the NonPersistentIdentifier when we are doing relocatable code where we need a persistent identifier?